test/conftest.py

```diff
@@ -28,7 +30,18 @@ def config() -> Generator:
     # vars are set
     from StaticNarrative.StaticNarrativeServer import get_config
 
-    yield generate_config(get_config())
+    original_conf = get_config()
+    assert original_conf is not None
+
+    # use a temp directory for the scratch and static file root dirs
+    with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
Just FYI, there's a built-in `tmp_path` fixture.
https://docs.pytest.org/en/6.2.x/reference.html#std-fixture-tmp_path

This is fine, as it flows logically.